core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -54,7 +54,7 @@ import scala.util.control.ControlThrowable
  *       1. use new producer with following settings
  *            acks=all
  *            retries=max integer
- *            block.on.buffer.full=true
+ *            max.block.ms.config=max long
```

Comment:
Thanks. Yes, but those variables are part of the public API and we cannot remove them without going through a deprecation cycle. It would have to be done in a separate PR and I'm not sure what is the current policy with regards to things like that (if we think it's worth doing or not).